spyder/plugins/ipythonconsole/widgets/debugging.py

```diff
@@ -368,6 +418,39 @@ def reset(self, clear=False):
                            password=password)
 
     # --- Private API --------------------------------------------------
+    def _toggle_pdb_single_letter_mode(self):
+        if not self.is_debugging():
+            return
+        if self._pdb_single_letter_mode:
+            self._pdb_single_letter_mode = False
+            prompt = self._current_prompt()
+        else:
+            self._pdb_single_letter_mode = True
+            prompt = self._pdb_single_letter_prompt
+        if self._reading:
+            self._update_pdb_prompt(prompt)
+            self._show_prompt(prompt)
+
+    def _current_prompt(self):
+        return "pdb [{}]: ".format(self._pdb_history_input_number + 1)
```

Comment:
My suggestions for the Pdb prompt are `IPdb [n]:` or `In (dbg) [n]:` (I think I'm more in favor of the first one).